.pre-commit-config.yaml

```diff
@@ -7,6 +7,13 @@ exclude: |
   )$
 
 repos:
+  - repo: local
```

Comment:
Could we use https://github.com/astral-sh/uv-pre-commit instead which ensures that `uv` gets installed if it isn't present on the system